### DISCUSSION TITLE
Incubation into Apache Software Foundation as XTable

### DISCUSSION THREAD

**Discussion by kywe665** [apache/incubator-xtable#356] [General]
Dropping a quick note here in case anyone is wondering about the project progress. We are starting incubation into the Apache Software Foundation, and you may notice the GitHub repo is now under https://github.com/apache/incubator-xtable/. The project will be renamed as XTable and all ASF processes and procedures will be adopted soon. Here is an overview of what is in progress: https://incubator.apache.org/projects/xtable.html

If you want to contribute to this incubation process or in any other way to the project, chime in on github here and there is also now a [mailing list you can join](mailto:dev-subscribe@xtable.apache.org) to engage in or follow community discussions.

**Comment by alberttwong** [comment]
Congrats!
